src/tests/unit/services/YotiService.test.ts

```diff
@@ -494,6 +500,33 @@ describe("YotiService", () => {
 			expect(statusCode).toBe(HttpCodesEnum.OK);
 		});
 
+		it("should include post office name if include in request", async () => {
```

Comment:
do we have the reverse test as well - it should use the default post office name if not present in the request?

Reply:
The [existing test](https://github.com/alphagov/di-ipv-cri-f2f-api/blob/456e491a5952c272e6477fd661961651664d16ce/src/tests/unit/services/YotiService.test.ts#L484C16-L484C16) which I've not changed already checks for the only request payload (without name being passed in) and checks the Yoti request includes the hardcoded value